partials/about_nav.html

```diff
@@ -14,7 +14,6 @@
                 <a href="{{item.url.path}}"><h5>{{item.title}}</h5></a></li>
             </li>
         {% endfor %}
-        <li><a href="{{g.static('/source/docs/CP_Vogelstein.pdf').url.path}}" target="_blank" rel="noopener"><h5>Funding</h5></a></li>
```

Comment:
this line also delete: https://github.com/neurodata/neurodata.io/pull/245/files#diff-dad88da953fd09c2c8b366b95e4c44b3R24

Reply:
done